src/api/java/blusunrize/immersiveengineering/api/crafting/builders/MineralMixBuilder.java

```diff
@@ -111,4 +115,38 @@ public MineralMixBuilder addOre(IngredientWithSize ingredient, float chance, ICo
 		oresArray.add(jsonObject);
 		return this;
 	}
+
+	public MineralMixBuilder addOverworldSpoils()
+	{
+		JsonObject gravel = new JsonObject();
+		gravel.addProperty("chance", 0.2f);
+		gravel.add("output", serializeItemStack(new ItemStack(Items.GRAVEL)));
+		spoilsArray.add(gravel);
+		JsonObject cobblestone = new JsonObject();
+		cobblestone.addProperty("chance", 0.5f);
+		cobblestone.add("output", serializeItemStack(new ItemStack(Items.COBBLESTONE)));
+		spoilsArray.add(cobblestone);
+		JsonObject deepslateCobblestone = new JsonObject();
+		deepslateCobblestone.addProperty("chance", 0.3f);
+		deepslateCobblestone.add("output", serializeItemStack(new ItemStack(Items.COBBLED_DEEPSLATE)));
+		spoilsArray.add(deepslateCobblestone);
+		return this;
+	}
+
+	public MineralMixBuilder addNetherSpoils()
+	{
+		JsonObject netherrack = new JsonObject();
+		netherrack.addProperty("chance", 0.5f);
+		netherrack.add("output", serializeItemStack(new ItemStack(Items.NETHERRACK)));
+		spoilsArray.add(netherrack);
+		JsonObject basalt = new JsonObject();
+		basalt.addProperty("chance", 0.3f);
+		basalt.add("output", serializeItemStack(new ItemStack(Blocks.BASALT.asItem())));
+		spoilsArray.add(basalt);
+		JsonObject gravel = new JsonObject();
+		gravel.addProperty("chance", 0.2f);
+		gravel.add("output", serializeItemStack(new ItemStack(Blocks.BLACKSTONE.asItem())));
+		spoilsArray.add(gravel);
```

Comment:
Either this field is named wrong, or its output is wrong.

Is there are reason you went for blackstone instead of gravel? I feel like blackstone is just a little "too nice" for that. Plus gravel is what's used in the Archaic Digsite as a "spoil" (before this system existed).

Reply:
Blackstone seemed more "nether-y" but I can change it to gravel! And yes, I was copying code and forgot to remove the field XD